rules-tests/Php80/Rector/Class_/AnnotationToAttributeRector/Fixture/Doctrine/doctrine_nested_attributes.php.inc

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Tests\Php80\Rector\Class_\AnnotationToAttributeRector\Fixture;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table()
+ */
+class UserGroup
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue(strategy="IDENTITY")
+     * @ORM\Column(type="integer")
+     */
+    private int $id;
+
+    /**
+     * @ORM\ManyToMany(targetEntity="User")
+     * @ORM\JoinTable(name="user_group_user",
+     *      joinColumns={@ORM\JoinColumn(name="user_group_id", referencedColumnName="id")},
+     *      inverseJoinColumns={@ORM\JoinColumn(name="user_id", referencedColumnName="id")}
+     *      )
+     */
+    private $users;
+}
+
+?>
+-----
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Tests\Php80\Rector\Class_\AnnotationToAttributeRector\Fixture;
+
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity]
+#[ORM\Table]
+class UserGroup
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue(strategy: 'IDENTITY')]
+    #[ORM\Column(type: 'integer')]
+    private int $id;
+    #[ORM\ManyToMany(targetEntity: 'User')]
+    #[ORM\JoinTable(name: 'user_group_role')]
+    #[ORM\JoinColumn(name: 'user_group_id', referencedColumnName: 'id')]
+    #[ORM\InverseJoinColumn(name: 'user_id', referencedColumnName: 'id')]
```

Comment:
The join table has support for nested attributes. To keep it 1:1 to original code, we should provide it here too:
See https://www.doctrine-project.org/2022/01/11/orm-2.11.html
Most people would probably go to this version, because older PHP/Doctrine didn't allow it technically, then tried to move back to original nesting formatting, as it makes more sense.

It's also quite complex case to support and current doc parser does not allow it :)